src/drivers/mpu6000/mpu6000.h

```diff
@@ -204,12 +204,12 @@
 #define MPU6000_ACCEL_DEFAULT_DRIVER_FILTER_FREQ	30
 
 #define MPU6000_GYRO_DEFAULT_RANGE_G				8
-#define MPU6000_GYRO_DEFAULT_RATE					1000
+#define MPU6000_GYRO_DEFAULT_RATE					8000
```

Comment:
OK, what would the impact of this be. Is there a way for me to check performance?